### ISSUE TITLE
User experience - Where's the screencast?

### ISSUE THREAD

**Issue by captproton** [ryanb/railscasts#11]
There should be at least an image of a player for the user to click on, IMHO.

**Comment by captproton** [comment]
This might be already in the works, it takes a moment to realize where the user is supposed to click to play the video.  Overall, I like the update.  Very impressive.

**Comment by ryanb** [comment]
I just released some design tweaks and am wondering if it addresses this problem. Now if you click on the Watch Episode button the screencast will start playing immediately. If you get to an episode through other means (such as clicking on the title) it will display a "Click to Play Video" button. Let me know what you think.

**Comment by captproton** [comment]
The action resulting from the "Watch Episode" is good.  I like it.  The blue button bar on each episode's page makes where to click obvious.  Why did you choose that over the traditional screenshot with a triangle on top?

**Comment by mindbreaker** [comment]
The blue button is a good solution. But i think for the usability it will be nice, to display a transparent image if the video is not playing. And it would be nice to pause the video with "space" or with a click on it. So i don't have to scroll down and press the pause button.

**Comment by captproton** [comment]
The blue bar works, I was just wondering why to step away from conventions.  The usability is a good point.

**Comment by ryanb** [comment]
@captproton, I switched to a blue bar to reduce the vertical height. Some people come to the episode page and are not interested in the video. If I showed a screenshot it would make them scroll down the 600 pixels to get to the show notes/comments.

@mindbreaker, Thanks for the suggestion about the space bar to play/pause the video. I can probably implement that with JavaScript, but I don't know how to get the same behavior for clicking the video since that is handled by SublimeVideo. I'll see if they have suggestions though.
